controllers/users.js

```diff
@@ -212,14 +203,14 @@ const getUsernameAvailabilty = async (req, res) => {
  * @param res {Object} - Express response object
  */
 
-const getSelfDetails = (req, res) => {
+const getSelfDetails = async (req, res) => {
```

Comment:
since you are making this function `async` can you please confirm if the function callee is using await `keyword`.